### PR TITLE
Integration features for 7.14

### PR DESCRIPTION
**Pull request by joostverhoog**
Documentation of integration features for release 7.14

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
@MarkvanMents Hi Mark, I reviewed "Published REST Operation." Please review the other docs and publish for 7.14. Cheers.

File: content/refguide/published-rest-service-json-schema.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+parent: "published-rest-services"
+---
+
+{{% alert type="info" %}}
+
+Support for *JSON Schema* was added in version 7.14.0.
+
+{{% /alert %}}
+
+## 1 Introduction
```
Comment (MarkvanMents) [inline content/refguide/published-rest-service-json-schema.md]: Does this introduction describe clearly what the schema is and what the document does?

Comment (joostverhoog) [reply content/refguide/published-rest-service-json-schema.md]: I've improved it

File: content/refguide/published-rest-service-json-schema.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+```json
+{ 
+  "type": "array",
+  "items": [{ "$ref": "#/definitions/#definition_name#"}]
+}
+```
+
+When the microflow returns a primitive, the schema depends on the type:
+
+| Microflow result | Schema      |
```
[the remark this answers is not in the document]
Comment (joostverhoog) [reply content/refguide/published-rest-service-json-schema.md]: Yes, it's correct. For instance `2018-04-25` is a valid date, but it's not valid JSON. Therefore the more generic type "file" is used.
An entity with a date attribute will output `{ "date": "2018-04-25" }` which is valid json

File: content/refguide/published-rest-service-json-schema.md (hunk 1 above)
Comment (MarkvanMents) [reply content/refguide/published-rest-service-json-schema.md]: Thanks

**Pull request merged**